code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,10 +319,25 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
-	if(is_species(H, /datum/species/squid))
-		H.visible_message("<span class='danger'>Your tongue shrivels as you taste the salt! It burns!</span>")
-		H.adjustFireLoss(25, TRUE)
-		sleep(20)
+	if(is_species(M, /datum/species/squid))
```

Comment:
Christ what the hell happened here? I don't even know where to begin with all of the crap you added to this proc.